front/prompt/global_agent_helper_prompt.md

```diff
@@ -76,19 +76,20 @@ To synchronize Slack channels with Dust:
 
 - Inside `Admin` > `Connections` > `Slack`;
 - Navigate to the Slack data source settings;
-- Select the Slack channels you want to synchronize with Dust;
+- Select the Slack channels you want to synchronize with Dust- Dust doesn't synchronize private channels;
```

Comment:
Correct. However, to be extra clear since users can 1. not be very clear about what 'synchronizing' means and 2. be afraid that we access 'private' channels, I suggest adding

=== Just before "To synchr. slack channels ===
**Synchronizing** a channel means:
- that it will be available for data retrieval assistants;
- that the data accessible to assistants in the channel will be maintained up to date automatically by Dust.

=== Right after "- The dust bot will..."
Dust does **not** synchronize private channels. You can call the dust bot in private channels, and at the time you call it it will be able to access the message history. But this message history is immediately forgotten, and dust does not make the private channel data available for retrieval by other assistants.

Reply:
I think then, the best solution is to add "Synchronizing" as a General Concept right after "Connections & Data Sources"

Synchronizing
Synchronizing data with Dust means Dust will extract it and that the data will be available for data retrieval by the assistants.

Reply:
Ok for 2. Can you add this? 
Dust does not synchronize private channels. You can call @dust in private channels, it it will be able to access the message history to answer but the message history will be immediately forgotten. Dust does not make the private channel data available for retrieval by other assistants.